templates/zerver/for/open-source.md

```diff
@@ -1,104 +1,92 @@
-## Great for developers
-
-* **Format code as code:** Messages take Markdown formatting,
-  [including code
-  blocks](/help/format-your-message-using-markdown#code), making it
-  super easy to discuss some code or paste an error message.
-
-* **Clear permalinks:** Have a design discussion in chat and want to
-  link to it in your bug tracker or a commit message?  [Two
-  clicks](/help/share-a-message-or-conversation) makes a permanent
-  link.  The URL transparently shows where the discussion happened,
-  and for everyone who clicks through, Zulip's topic threading
-  provides a focused view on that conversation.
-
-* **Easy custom linkification:** Want to refer to issues or code
-  reviews with short notation like `#1234`, `T1234` — or any other
-  style of your own?  Set up any regex as a [custom linkification
-  filter](/help/add-a-custom-linkification-filter) for your
-  organization.
-
-* **Plays well with others:** With [60+ integrations](/integrations)
-  including GitHub, Travis CI, JIRA, and Stripe, chances are Zulip
```

Comment:
The integrations might be one more point to keep a mention of on the /for/open-source page -- I think most software projects will want some integrations, and they might be glad to see the names of things like GitHub and Travis CI. (Maybe just mention those two -- I just took another pass through the integrations list.)